android-test/build.gradle

```diff
@@ -47,6 +47,6 @@ dependencies {
   androidTestImplementation project(':okhttp-logging-interceptor')
   androidTestImplementation 'com.android.support.test:runner:1.0.2'
   androidTestImplementation 'com.android.support.test.espresso:espresso-core:3.0.2'
-  androidTestImplementation 'com.squareup.moshi:moshi:1.8.0'
-  androidTestImplementation 'com.squareup.moshi:moshi-kotlin:1.8.0'
+  androidTestImplementation 'com.squareup.moshi:moshi:1.9.2'
+  androidTestImplementation 'com.squareup.moshi:moshi-kotlin:1.9.2'
```

Comment:
If you are serializing Kotlin this is a potential behavior change. So make sure you tested.